### PR TITLE
Create UI movie detail screen

### PR DESCRIPTION
**Pull request by tinobk**

### CODE REVIEW

File: app/src/main/java/com/sun/tino/hottrailers/ui/movie_detail/MovieDetailActivity.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.sun.tino.hottrailers.ui.movie_detail;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
```
Comment (hieunn-0931) [inline app/src/main/java/com/sun/tino/hottrailers/ui/movie_detail/MovieDetailActivity.java:3]: format code

File: app/src/main/java/com/sun/tino/hottrailers/ui/movie_detail/YoutubeFragment.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    private YouTubePlayer mYouTubePlayer;
+    private String mTrailerId;
+
+    @Override
+    public void onCreate(Bundle bundle) {
+        super.onCreate(bundle);
+        initialize(BuildConfig.YOUTUBE_API_KEY, this);
+    }
+
+
```
Comment (hieunn-0931) [inline app/src/main/java/com/sun/tino/hottrailers/ui/movie_detail/YoutubeFragment.java]: remove bl

File: app/src/main/res/layout/fragment_movie_info.xml [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+                android:id="@+id/view_content"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:padding="@dimen/dp_4"
+                tools:context=".ui.movie_detail.info.MovieInfoFragment">
+
+                <de.hdodenhof.circleimageview.CircleImageView
+                    android:id="@+id/image_movie"
+                    android:layout_width="100dp"
+                    android:layout_height="100dp"
```
Comment (hieunn-0931) [inline app/src/main/res/layout/fragment_movie_info.xml]: hard code

File: app/src/main/res/layout/fragment_movie_info.xml [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <View
+                    android:id="@+id/view_one"
+                    android:layout_width="0dp"
+                    android:layout_height="0dp"
+                    android:layout_marginStart="4dp"
+                    android:layout_marginLeft="4dp"
+                    android:layout_marginTop="4dp"
+                    android:layout_marginEnd="8dp"
```
Comment (hieunn-0931) [inline app/src/main/res/layout/fragment_movie_info.xml]: hard code

File: app/src/main/res/layout/fragment_movie_info.xml [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+                </View>
+
+                <TextView
+                    android:id="@+id/text_name"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="4dp"
+                    android:layout_marginLeft="4dp"
+                    android:layout_marginTop="8dp"
+                    android:layout_marginEnd="4dp"
```
Comment (hieunn-0931) [inline app/src/main/res/layout/fragment_movie_info.xml]: hard code

**Review by hieunn-0931 — APPROVED** [review]

File: app/src/main/res/layout/activity_movie_detail.xml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        tools:context=".ui.movie_detail.MovieDetailActivity">
+
+        <androidx.appcompat.widget.Toolbar
+            android:id="@+id/toolbar"
+            android:layout_width="0dp"
+            android:layout_height="?actionBarSize"
+            android:layout_marginTop="@dimen/dp_24"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"></androidx.appcompat.widget.Toolbar>
```
Comment (vietanhk-0863) [inline app/src/main/res/layout/activity_movie_detail.xml]: self close

**Review by vietanhk-0863 — APPROVED** [review]

**Review by tuananhd-1813 — APPROVED** [review]

**Pull request merged**